types/bytebuffer/index.d.ts

```diff
@@ -3,6 +3,7 @@
 // Definitions by: Denis Cappellin <https://github.com/cappellin>
```

Comment:
My GitHub address is bad, cause it misses a "d". Can you fix it, please? 
    
    // Definitions by: Denis Cappellin <https://github.com/dcappellin>

Reply:
I would say to make it in separate PR. I have spotted it as well.